<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCanTx</class>
 <widget class="QWidget" name="FormCanTx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chEnable">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leId">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="inputMask">
        <string>HHHHHHHH</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QLineEdit" name="leData0">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData1">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData2">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData3">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData4">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData5">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData6">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData7">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="leLength">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>32</height>
        </size>
       </property>
       <property name="inputMask">
        <string>HH</string>
       </property>
       <property name="text">
        <string>08</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFormat">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbType">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>TX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>DEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbInterval">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
